### DISCUSSION TITLE
fresh install web shows only empty page

### DISCUSSION THREAD

**Discussion by GPadierna1** [inventree/InvenTree#6609] [General]
Hello,
Sorry to bother you people with what is for sure a nuance, but it's starting to driving me crazy. 
I've just installed (several times) Inventree (0.13.10) using docker (modified the 3 files: .env, docker-compose.yaml and Caddyfile accordingly), install finishes apparently good, no error messages. Then I put in my browser http://my_local_ip and it just loads a blank page, nothing else. No error either.
https://<localhost_ip> returns 'Connection refused' error. 

I'm following the instructions provided here: https://docs.inventree.org/en/latest/start/docker_install/
Is there something that I'm missing? 
Thanks

**Comment by matmair** [comment]
We will not be able to help you with this amount of information please either fill out an [install issue](https://github.com/inventree/InvenTree/issues/new?assignees=&labels=question%2Ctriage%3Anot-checked%2Csetup&projects=&template=install.yaml) or provide us the exact steps you did, the logs you get from all the setup steps you need to run and the proxy and server logs from when you load the local ip.

**Reply by GPadierna1** [reply]
I'm aware I didn't provide information helpful for troubleshooting. Basically just wanted the confirmation that I wasn't missing something from the installation process, maybe something that might seem so obvious that's not even in the instructions.
I'll fill out the install issue form you provided. Thanks!

**Comment by SchrodingersGat** [comment]
You should navigate to `http://inventree.localhost` by default, unless you have explicitly changed the `INVENTREE_SITE_URL` setting

**Reply by GPadierna1** [reply]
I did change the INVENTREE_SITE_URL in .env. But I still should be able access the site by entering 'localhost' or '127.0.0.1' in a browser on the machine where the docker containers are running. 
For instance, `curl localhost:80` (run from my the machine running the docker images) returns nothing (return code 0) and if run with `-v`, the output is this:
```
*   Trying 127.0.0.1:80...
* Connected to localhost (127.0.0.1) port 80 (#0)
> GET / HTTP/1.1
> Host: localhost
> User-Agent: curl/7.81.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Server: Caddy
< Date: Fri, 01 Mar 2024 07:39:50 GMT
< Content-Length: 0
< 
* Connection #0 to host localhost left intact
```
In the  'inventree-proxy' container logs, when I try to access it from a browser in the local machine, this is what's displayed:
![2024-03-01 08_35_27-inventree_blank_page](https://github.com/inventree/InvenTree/assets/14176274/bc383849-5a3b-41ee-bf17-f2817ccd25a1)
In text:
```
{"level":"info","ts":1709277705.3900928,"logger":"http.log.access","msg":"handled request","request":{"remote_ip":"172.22.0.1","remote_port":"47150","client_ip":"172.22.0.1","proto":"HTTP/1.1","method":"GET","host":"127.0.0.1","uri":"/","headers":{"Accept-Encoding":["gzip, deflate, br"],"Dnt":["1"],"Sec-Fetch-Site":["none"],"Sec-Fetch-User":["?1"],"User-Agent":["Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0"],"Connection":["keep-alive"],"Upgrade-Insecure-Requests":["1"],"Sec-Fetch-Dest":["document"],"Sec-Fetch-Mode":["navigate"],"Accept":["text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8"],"Accept-Language":["es-ES,es;q=0.8,en-US;q=0.5,en;q=0.3"]}},"bytes_read":0,"user_id":"","duration":0.000005168,"size":0,"status":0,"resp_headers":{"Server":["Caddy"]}}
{"level":"info","ts":1709277705.45323,"logger":"http.log.access","msg":"handled request","request":{"remote_ip":"172.22.0.1","remote_port":"47150","client_ip":"172.22.0.1","proto":"HTTP/1.1","method":"GET","host":"127.0.0.1","uri":"/favicon.ico","headers":{"Accept":["image/avif,image/webp,*/*"],"Accept-Encoding":["gzip, deflate, br"],"Dnt":["1"],"Sec-Fetch-Site":["same-origin"],"User-Agent":["Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0"],"Referer":["http://127.0.0.1/"],"Sec-Fetch-Dest":["image"],"Sec-Fetch-Mode":["no-cors"],"Accept-Language":["es-ES,es;q=0.8,en-US;q=0.5,en;q=0.3"],"Connection":["keep-alive"]}},"bytes_read":0,"user_id":"","duration":0.000010407,"size":0,"status":0,"resp_headers":{"Server":["Caddy"]}}
```
And no logs are being displayed on the other containers (worker, server and db) when I attempt and http browser access.

**Reply by matmair** [reply]
You are not able to access the full instance under domains other than the one listed in the enviroment file. That traffic is ignored. Where did you read that this should be possible?

**Reply by SchrodingersGat** [reply]
> I did change the INVENTREE_SITE_URL in .env. But I still should be able access the site by entering 'localhost' or '127.0.0.1' in a browser on the machine where the docker cont

What did you change it to? That is the *only* host that the caddy server will listen on
